apps/admin/src/components/proposalCards/Unsponsored.tsx

```diff
@@ -109,6 +109,18 @@ export const Unsponsored = ({
       ? true
       : 'You are not connected to the same network as the DAO';
 
+  const notDelegating = useMemo(() => {
```

Comment:
Just a suggestion based on my preference but definitely not required
  const isNotDelegating = useMemo(() => {
```